src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,140 @@
+const MetaTrader          = require('../../../user/metatrader/metatrader');
+const getCurrencyFullName = require('../../../../common/currency').getCurrencyFullName;
+const Client              = require('../../../../base/client');
+const BinarySocket        = require('../../../../base/socket');
+const localize            = require('../../../../../_common/localize').localize;
+const Url                 = require('../../../../../_common/url');
+const isCryptocurrency    = require('../../../../../_common/base/currency_base').isCryptocurrency;
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $form,
+        $txt_other_reason,
+        $closure_loading,
+        $closure_container,
+        $success_msg,
+        $error_msg,
+        $crypto,
+        $fiat;
+
+    const onLoad = () => {
+        $txt_other_reason  = $('#other_reason');
+        $closure_loading   = $('#closure_loading');
+        $closure_container = $('#closure_container');
+        $success_msg       = $('#msg_main');
+        $error_msg         = $('#msg_form');
+        $crypto            = $('#crypto');
+        $fiat              = $('#change-fiat');
+        $form              = $(form_selector);
+
+        BinarySocket.wait('landing_company').then(() => {
+            const currencies = Client.currentLandingCompany().legal_allowed_currencies;
+            const current_currency = Client.get('currency');
+            const show_currencies = currencies.filter((val) => val !== current_currency);
+
+            if (show_currencies) {
+                show_currencies.forEach((currency) => {
+                    if (isCryptocurrency(currency)) {
+                        $crypto.find('ul').append(`<li>${getCurrencyFullName(currency)}</li>`);
+                    } else {
+                        $fiat.find('ul').append(`<li>${getCurrencyFullName(currency)}</li>`);
+                    }
+                });
+                if (!$crypto.find('li').length) $crypto.setVisibility(0);
+                if (!$fiat.find('li').length) $fiat.setVisibility(0);
+            }
+            if (!MetaTrader.isEligible()) {
+                $('#mt5_withdraw').setVisibility(0);
+            }
+        }).catch((error) => {
+            showFormMessage(error.message);
+        });
+        $closure_container.setVisibility(1);
+
+        $(form_selector).on('submit', (event) => {
+            event.preventDefault();
+            submitForm();
+        });
+
+        $txt_other_reason.on('keyup', () => {
+            const input = $txt_other_reason.val();
+            if (input && (input.length < 5 || input.length > 250)) {
+                $txt_other_reason.addClass('error-field');
+                $error_msg
+                    .addClass('error-field')
+                    .html('The reason should be between 5 and 250 characters')
```

Comment:
this should be `.text(` instead, there are no html tags in the string to need a `.html(`. also you need a `localize(` wrapper around the text. so it should be `.text(localize('The reason should be between 5 and 250 characters'))`

Reply:
Oh alright 👍